Models.py

```diff
@@ -0,0 +1,66 @@
+"""
+Define all data models.
+"""
+
+from py2neo.ogm import (GraphObject, Property, Related, RelatedTo, RelatedFrom)
+
+
+class Person(GraphObject):
+    __primarykey__ = "uid"
+
+    # Properties.
+    username = Property()
+    display_name = Property()
+    uid = Property()
+    email = Property()
+    photo = Property()
+
+    # Relationships.
+    Knows = Related("Person", "KNOWS")
+    Circles = RelatedTo("Circle", "PART_OF")
+    InvitedTo = Related("Event", "INVITED_TO")
+
+    def __init__(self, username, display_name, email, photo):
+        self.username = username
+        self.display_name = display_name
+        self.email = email
+        self.photo = photo
+
+class Circle(GraphObject):
+    __primarykey__ = "cid"
+
+    # Properties.
+
+    name = Property()
+    cid = Property()
+
+    # Relationships.
+    Contains = Related("Person", "CONTAINS")
+    Scheduled = Related("Event", "SCHEDULED")
+
+    def __init__(self, name, members, events):
+        self.name = name
+        self.cid = 000  # How to determine this?
```

Comment:
See above comment. We shouldn't be managing unique id's ourselves, let's leave it to neo4j